examples/due_date_probabilities/probability_estimation.py

```diff
@@ -125,9 +125,9 @@ def raw_probabilities(raw_data: str) -> pd.DataFrame:
 
 def resampled(raw_probabilities: pd.DataFrame) -> List[int]:
     sample_data = []
-    for index, row in raw_probabilities.iterrows():
+    for _, row in raw_probabilities.iterrows():
         count = row.probability * 1000
-        for i in range(int(count)):
+        for _i in range(int(count)):
```

Comment:
If you insist,
```suggestion
        for __ in range(int(count)):
```

I'd do it differently, something like:
```python
    sample_data = []
    for _, row in raw_probabilities.iterrows():
        count = row.probability * 1000
        sample_data.extend([row.days] * int(count))
```

Reply:
It's actually the ruff fix suggests `_i` to have it not complain.

Reply:
we have it there for legibility.

Reply:
but sure that's cleaner.